mobile/src/bundles/common/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,76 @@
+import { type ImageStyle, type StyleProp } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import { getAvatarInitials } from 'shared/build/helpers/helpers';
```

Comment:
Seems like the wrong path: 'shared/build/helpers/helpers';

Reply:
fix